components/studio/bin/hab-studio.sh

```diff
@@ -1236,7 +1064,7 @@ hab="$libexec_path/hab"
 # The author of this program
 author='@author@'
 # The short version of the program name which is used in logging output
-program=$($bb basename $0)
+program=$($bb basename "$0")
```

Comment:
If `$0` is quoted, won't shellcheck also complain about `$($bb basename "$0")` being unquoted?

Reply:
It doesn't. I'm not exactly sure why. It would be fine to quote the `$(…)` as well, but I haven't been quoting where `shellcheck` doesn't seem to thing it's necessary. Quoting both is probably better for correctness. It's not likely that `$0` would contain a any `$IFS` characters, but you never know.